src/services/types.ts

```diff
@@ -583,6 +583,8 @@ export interface LanguageService {
      * @param position A zero-based index of the character where you want the quick info
      */
     getQuickInfoAtPosition(fileName: string, position: number): QuickInfo | undefined;
+    /** @internal */

```

Comment:
@gabritto would it be possible to make this type public?

Reply:
This isn't really a stable API at the moment, that's why it's marked internal. Why would you like this to be public?

Reply:
I'm working on a tool that uses CodeMirror 6 with TypeScript in a web worker, and there are various places where I'd love to have a bit more control over how things unfold. For example, I'd like to limit the verbosity of quick info shown in hover tooltips, but I also have some places where I'd like to give users an explicit way to increase verbosity in views created specifically for that purpose. I've quickly tested this feature, and it works perfectly so it would be awesome to have it accessible via the public API.